shock-server.cfg

```diff
@@ -0,0 +1,56 @@
+[Address]
+# IP and port for api
+# Note: use of port 80 may require root access
+# 0.0.0.0 will bind Shock to all IP's
+api-ip=0.0.0.0
+api-port=7044
+
+[Admin]
+email=kkeller@lbl.gov
+users=ciservices
+
+[Anonymous]
+# Controls an anonymous user's ability to read/write
+# values: true/false
+read=true
+write=true
+create-user=true
+
+[Auth]
+; for auth2
+globus_token_url=https://ci.kbase.us/services/auth/api/legacy/globus/goauth/token?grant_type=client_credentials
```

Comment:
Would Travis use these services, or would it spin up its own auth2 service for testing?